modules/javafx.controls/src/test/java/test/javafx/scene/control/ControlPropertiesTest.java

```diff
@@ -95,12 +94,14 @@
  * Perhaps the test should scan classpath and find all the Controls automagically.
  */
 public class ControlPropertiesTest {
-    private static final boolean FAIL_FAST = true;
 
+    private static final boolean FAIL_FAST = !true;
```

Comment:
Did you intend to commit this?  It was `true` before which seems good for tests, otherwise I'd really suggest using `false` here.

Reply:
sorry, committed by accident.  thanks for noticing!